### PR TITLE
Update pycocotools version

### PR DESCRIPTION
**Pull request by dkupnicki**
CI was failing while installing pycocotools. This PR fixes that

### CODE REVIEW

File: setup_deb.sh [hunk 1]
```diff
@@ -140,6 +140,10 @@ pip3 install --no-deps --upgrade \
    inflect==6.0.4 \
    pydantic==1.10.9
 
+git clone https://github.com/cocodataset/cocoapi.git
```
Comment (jan-grzybek-ampere) [inline setup_deb.sh:143]: hmm, no way to install from pypi? can you point me to the CI issue? fyi this repo is no longer maintained - there is a fork which is and with which pypi packages are being built: https://github.com/ppwwyyxx/cocoapi

**Review by jan-grzybek-ampere — CHANGES_REQUESTED** [review]
Straightforward solution is just to move pycocotools to 2.0.6 from 2.0.4. 
We need to find better way to manage all these dependencies. I will look into that sometime soon.

File: setup_deb.sh (hunk 1 above)
Comment (dkupnicki) [reply setup_deb.sh:143]: You can click on one of the red crosses on the commits above and the issue will be there.

**Comment by dkupnicki** [comment]
Yep, changing the package version is definitely a much better solution to this problem 😀

**Review by jan-grzybek-ampere — APPROVED** [review]

**Pull request merged**